test/jdk/java/lang/reflect/code/bytecode/TestSmallCorpus.java

```diff
@@ -144,6 +146,8 @@ private void testRoundTripStability(Path path) throws Exception {
                     stats[4] += ca.maxLocals();
                     stats[5] += ca.maxStack();
                 }
+            } catch (UnsupportedOperationException uoe) {
+                // InvokeSuperOp
```

Comment:
Is this related to bytecode instructions for construction not conforming to an expected shape?

Reply:
Ok, i see now. Your assumptions are correct.
(Note #225 lumps `InvokeSuperOp` into `InvokeOp` but it does not change the assumptions.)